grafana-dashboard/README.md

```diff
@@ -0,0 +1,23 @@
+# Open Policy Agent Grafana dashboard
+
+![](images/dashboard-image-1.png)
+
+This folder hosts an example on the Grafana dashboard for Open Policy Agent.
+
+It visualizes most of the metrics recorded by Open Policy Agent as documented in 
+https://www.openpolicyagent.org/docs/latest/monitoring/
+
+The version of OPA that this is made for is 0.26.
```

Comment:
I would either skip this or at least add “or above”. Given how v0.27.0 was released yesterday this looks outdated before it’s even published :)

Reply:
Thanks for the feedback @anderseknert . Now I am curious how do we got updated when new versions are out 😭 Do you follow a mailing list or some sort of tech blogs?

Reply:
The best way to stay up to date with everything going on in the OPA community is to join the [OPA Slack](https://slack.openpolicyagent.org/). But if you only want to be notified of when new releases are pushed you can ask Github to notify you of that. Simply press the "Watch" button up to the right and then choose "Custom" followed by "Releases".